stdlib/streams/src/main/ballerina/streams/snapshot.bal

```diff
@@ -0,0 +1,291 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerina/math;
+import ballerina/runtime;
+import ballerina/task;
+import ballerina/config;
+import ballerina/internal;
+import ballerina/time;
```

Comment:
Shall we order the imports alphabetically?

Reply:
fixed in 9f827ee. But shouldn't this be fixed in IDE plugin to do it automatically?